app/scripts/controllers/overview.js

```diff
@@ -1350,11 +1352,11 @@ function OverviewController($scope,
     }, {poll: true, pollInterval: DEFAULT_POLL_INTERVAL}));
 
     if ($scope.AEROGEAR_MOBILE_ENABLED) {
-      watches.push(DataService.watch({ group: "mobile.k8s.io", version: "v1alpha1", resource: "mobileclients" }, context, function (clients) {
+      watches.push(MobileClientsService.createWatcher(context, function(clients) {
```

Comment:
I would go with `MobileClientsService.watche` since we are just wrapping a mobile clients watch